101-clickable-grid/clickable-grid.html

```diff
@@ -0,0 +1,73 @@
+<!doctype html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <title>Document</title>
+    <style>
+        .content>div{
+            border:1px solid #ff3300;
+            width:1200px;
+            display:flex;
+            justify-content: left;
+            flex-wrap: wrap;
+        }
+        .content>div>div{
+            display: flex;
+            box-sizing:border-box;
+            margin:5px;
+            height:40px;
+            background:darkgoldenrod;
+            width:40px;
+
+
+        }
+    </style>
+</head>
+<body>
+<button>++</button>
+<span>1</span>
+<button>---</button>
+<div class="content">
+</div>
+<script>
+window.onload = function () {
+    var btns = document.querySelectorAll('button'),
+        addBtn = btns[0],
+        subBtn = btns[1],
```

Comment:
avoid using index directly to find buttons, you should give each button a class or an id